src/widget/widget.cpp

```diff
@@ -706,7 +706,7 @@ void Widget::setStatusMessage(const QString &statusMessage)
 void Widget::reloadHistory()
 {
     for (auto f : FriendList::getAllFriends())
-        f->getChatForm()->loadHistory(QDateTime::currentDateTime().addDays(-7), true);
+        f->getChatForm()->loadHistory(QDateTime::currentDateTime().addDays(-1), true);
```

Comment:
I was experimenting with code and changed this to see what happens, I forgot to change it back...